pkg/cclient/README.md

```diff
@@ -0,0 +1,36 @@
+# Compiling the C Library
+
+## Creating the .h and the .so files:
+
+
+```go build -o lib_udsclient.so -buildmode=c-shared cclient.go```
```

Comment:
```suggestion
```cmd 
go build -o lib_udsclient.so -buildmode=c-shared cclient.go
```
```
Again I'm surprised the linter didn't flag this